core/src/main/java/com/scalar/db/sql/SqlStatementSession.java

```diff
@@ -12,6 +16,19 @@
 
   ResultSet execute(Statement statement);
 
+  default ResultSet execute(BindableStatement<?> statement, Value... positionalValues) {
+    return execute(statement, Arrays.asList(positionalValues));
+  }
+
+  default ResultSet execute(BindableStatement<?> bindableStatement, List<Value> positionalValues) {
+    return execute(bindableStatement.bind(positionalValues));
+  }
+
+  default ResultSet execute(
+      BindableStatement<?> bindableStatement, Map<String, Value> namedValues) {
+    return execute(bindableStatement.bind(namedValues));
+  }
```

Comment:
I added these three `execute()` methods to execute statement with parameter values.

We can pass parameters as positional values or named values.